### ISSUE TITLE
Update to 1.14.0 produce error

### ISSUE THREAD

**Issue by ireinhart** [bugsnag/bugsnag-symfony#173]
### Describe the bug

With the update to 1.14.0 from 1.13.0 we got the following error log, if we build the app for the PROD app ENV.

```
Class Bugsnag\BugsnagBundle\DependencyInjection\Configuration located in ./vendor/bugsnag/bugsnag-symfony/DependencyInjection/configuration-without-return-type.php does not comply with psr-4 autoloading standard. Skipping.
Class Bugsnag\BugsnagBundle\DependencyInjection\Configuration located in ./vendor/bugsnag/bugsnag-symfony/DependencyInjection/configuration-with-return-type.php does not comply with psr-4 autoloading standard. Skipping.
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
92 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  
!!  Fatal error: Uncaught Error: Class "Bugsnag\BugsnagBundle\DependencyInjection\Configuration" not found in /app/vendor/bugsnag/bugsnag-symfony/DependencyInjection/BugsnagExtension.php:22
!!  Stack trace:
!!  #0 /app/vendor/symfony/dependency-injection/Compiler/MergeExtensionConfigurationPass.php(76): Bugsnag\BugsnagBundle\DependencyInjection\BugsnagExtension->load(Array, Object(Symfony\Component\DependencyInjection\Compiler\MergeExtensionConfigurationContainerBuilder))
!!  #1 /app/vendor/symfony/http-kernel/DependencyInjection/MergeExtensionConfigurationPass.php(39): Symfony\Component\DependencyInjection\Compiler\MergeExtensionConfigurationPass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
!!  #2 /app/vendor/symfony/dependency-injection/Compiler/Compiler.php(94): Symfony\Component\HttpKernel\DependencyInjection\MergeExtensionConfigurationPass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
!!  #3 /app/vendor/symfony/dependency-injection/ContainerBuilder.php(762): Symfony\Component\DependencyInjection\Compiler\Compiler->compile(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
!!  #4 /app/vendor/symfony/http-kernel/Kernel.php(594): Symfony\Component\DependencyInjection\ContainerBuilder->compile()
!!  #5 /app/vendor/symfony/http-kernel/Kernel.php(136): Symfony\Component\HttpKernel\Kernel->initializeContainer()
!!  #6 /app/src/Kernel.php(36): Symfony\Component\HttpKernel\Kernel->boot()
!!  #7 /app/vendor/symfony/framework-bundle/Console/Application.php(169): App\Kernel->boot()
!!  #8 /app/vendor/symfony/framework-bundle/Console/Application.php(75): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands()
!!  #9 /app/vendor/symfony/console/Application.php(149): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
!!  #10 /app/bin/console(42): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput))
!!  #11 {main}
!!    thrown in /app/vendor/bugsnag/bugsnag-symfony/DependencyInjection/BugsnagExtension.php on line 22
!!  PHP Fatal error:  Uncaught Error: Class "Bugsnag\BugsnagBundle\DependencyInjection\Configuration" not found in /app/vendor/bugsnag/bugsnag-symfony/DependencyInjection/BugsnagExtension.php:22
!!  Stack trace:
!!  #0 /app/vendor/symfony/dependency-injection/Compiler/MergeExtensionConfigurationPass.php(76): Bugsnag\BugsnagBundle\DependencyInjection\BugsnagExtension->load(Array, Object(Symfony\Component\DependencyInjection\Compiler\MergeExtensionConfigurationContainerBuilder))
!!  #1 /app/vendor/symfony/http-kernel/DependencyInjection/MergeExtensionConfigurationPass.php(39): Symfony\Component\DependencyInjection\Compiler\MergeExtensionConfigurationPass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
!!  #2 /app/vendor/symfony/dependency-injection/Compiler/Compiler.php(94): Symfony\Component\HttpKernel\DependencyInjection\MergeExtensionConfigurationPass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
!!  #3 /app/vendor/symfony/dependency-injection/ContainerBuilder.php(762): Symfony\Component\DependencyInjection\Compiler\Compiler->compile(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
!!  #4 /app/vendor/symfony/http-kernel/Kernel.php(594): Symfony\Component\DependencyInjection\ContainerBuilder->compile()
!!  #5 /app/vendor/symfony/http-kernel/Kernel.php(136): Symfony\Component\HttpKernel\Kernel->initializeContainer()
!!  #6 /app/src/Kernel.php(36): Symfony\Component\HttpKernel\Kernel->boot()
!!  #7 /app/vendor/symfony/framework-bundle/Console/Application.php(169): App\Kernel->boot()
!!  #8 /app/vendor/symfony/framework-bundle/Console/Application.php(75): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands()
!!  #9 /app/vendor/symfony/console/Application.php(149): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
!!  #10 /app/bin/console(42): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput))
!!  #11 {main}
!!    thrown in /app/vendor/bugsnag/bugsnag-symfony/DependencyInjection/BugsnagExtension.php on line 22
```

### Steps to reproduce
1. update bugsnag/bugsnag-symfony to 1.14.0

### Environment
* Bugsnag version: 1.14.0
* PHP version: 8.0.30
* Symfony version: 4.4.51

**Comment by MichaelRussell-Regit** [comment]
Experiencing the same problem: Symfony 5.4.34 (env: dev, debug: true)

**Comment by clr182** [comment]
Hi, 

We have released [v1.14.1 of the bugsnag-symfony notifier](https://github.com/bugsnag/bugsnag-symfony/releases/tag/v1.14.1). This update should fix the issues you are seeing. However,
If after upgrading your version you are still seeing these issues please let us know and we can investigate further.
